qiskit/pulse/channels.py

```diff
@@ -181,6 +181,11 @@ class ControlChannel(PulseChannel):
     prefix = 'u'
 
 
+class Frame(PulseChannel):
```

Comment:
The `PulseChannel` is correct superclass of frame? I think `Channel` and `Frame` are different classes though most of code may be duplicated.

Reply:
I'm not too keen on duplicating code.

Reply:
This loop inside the function doesn't allow multiprocessing. If the function is defined for single entry we can easily perform multiprocessing with another transformation.

Reply:
Sorry, my comment above is in a wrong place. You've already updated the code. Thanks.